cachito/web/models.py

```diff
@@ -948,6 +1015,12 @@ class PackageManager(db.Model):
     id = db.Column(db.Integer, primary_key=True)
     name = db.Column(db.String, nullable=False)
 
+    # Pairs of mutually exclusive package managers (locally - for the same subpath)
+    # This relation is automatically symmetric - (A, B) implies (B, A)
+    LOCALLY_EXCLUSIVE = [
+        ("npm", "yarn"),
+    ]
```

Comment:
Not sure if this class is the right place to declare this, WDYT?

Reply:
Out of all we have currently, this seems a reasonable location as it provides some context.

Reply:
Should this be in the config instead?

This way it would be easier to toggle the validation for git-submodules in the future. i.e, introduce the validation and remove it if need. So deployments won't need a hotfix.

Reply:
In my mind, there was no reason to change exclusivity settings without a corresponding code change, but having it in config would be more flexible. I'll change this.